src/Cards.java

```diff
@@ -0,0 +1,18 @@
+/**
+ * Created by strejolopez on 11/29/16.
+ */
+public class Cards {
+    private String title;
+    private String description;
+
+    public Cards(String title, String description){
+        this.title = title;
+        this.description = description;
+    }
+
+    public String getTitle() { return this.title; }
```

Comment:
For this our cohort lets go ahead and expand the getters and setters instead of putting the code all on one line.